runtime/src/main/java/org/corfudb/security/sasl/plaintext/PlainTextLoginModule.java

```diff
@@ -3,19 +3,19 @@
 import java.io.IOException;
 import java.util.Map;
 
+import javax.security.auth.Subject;
 import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
 import javax.security.auth.callback.NameCallback;
 import javax.security.auth.callback.PasswordCallback;
-import javax.security.auth.callback.CallbackHandler;
 import javax.security.auth.callback.UnsupportedCallbackException;
 import javax.security.auth.login.LoginException;
 import javax.security.auth.spi.LoginModule;
-import javax.security.auth.Subject;
 
 /**
  * Created by sneginhal on 01/27/2017
  *
- * Implementation of the plain text LoginMoodule.
+ * <p>Implementation of the plain text LoginMoodule.
```

Comment:
Ah interesting, didn't realize that. Since that's the style from the google style guide, let's go with that.